mongodbatlas/projects_test.go

```diff
@@ -219,7 +219,9 @@ func TestProject_Create(t *testing.T) {
 		}`)
 	})
 
-	project, _, err := client.Projects.Create(ctx, createRequest)
+	opts := &CreateProjectOptions{ProjectOwnerID: "1"}
```

Comment:
Can we add a test with `CreateProjectOptions` set to nil, and check if the URI is formed correctly in both too?

Reply:
sure

Reply:
done